src/components/taskForm.vue

```diff
@@ -0,0 +1,146 @@
+<template>
+  <div class="task-form-page__card">
+    <h2 class="task-form-page__card-title">{{ title }}</h2>
+    <v-form @submit.prevent="onSubmit" ref="formRef">
+      <div class="task-form-page__card-input">
+        <label for="task">Task:</label>
+        <input
+          id="task"
+          v-model="localTask"
+          type="text"
+          class="task-form-page__card-custom-input"
+          required
+        />
+      </div>
+      <div class="task-form-page__card-input">
+        <label for="estimatedTime">Estimate Time:</label>
+        <input
+          id="estimatedTime"
+          v-model="localEstimatedTime"
+          type="text"
+          class="task-form-page__card-custom-input"
+          required
+        />
+      </div>
+      <div class="task-form-page__card-button">
+        <v-btn
+          class="task-form-page__card-custom-button task-form-page__card-cancel-button"
+          @click="onCancel"
+          type="button"
+        >
+          Cancel
+        </v-btn>
+        <v-btn
+          class="task-form-page__card-custom-button task-form-page__card-save-button"
+          type="submit"
+        >
+          Save
+        </v-btn>
+      </div>
+    </v-form>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { ref, watch } from "vue";
+
+const props = defineProps<{
+  title: string;
+  task?: string;
+  estimatedTime?: string;
+}>();
+
+const emit = defineEmits<{
+  (e: "save", data: { task: string; estimatedTime: string }): void;
+  (e: "cancel"): void;
+}>();
+
+const localTask = ref(props.task || "");
+const localEstimatedTime = ref(props.estimatedTime || "");
```

Comment:
```suggestion
const localTask = ref("");
const localEstimatedTime = ref("");
```

Assign props values on page mounting